src/uk/gov/hmcts/contino/AppPipelineDsl.groovy

```diff
@@ -29,8 +29,9 @@ class AppPipelineDsl extends CommonPipelineDsl implements Serializable {
     WarningCollector.addPipelineWarning("deprecated_set_vault_name", "setVaultName() is deprecated, see https://github.com/hmcts/cnp-jenkins-library#secrets-for-functional--smoke-testing ", new Date().parse("dd.MM.yyyy", "27.08.2019"))
   }
 
-  void enableDbMigration() {
+  void enableDbMigration(String dbMigrationVaultName = "") {
```

Comment:
Should we make it a non-breaking change, just add it to pipelinewarnings ? something like : https://github.com/hmcts/cnp-jenkins-library/blob/a50fd5216c1b42064a9557e59b479509bff91d27/src/uk/gov/hmcts/contino/AppPipelineDsl.groovy#L29

Reply:
isn't this non breaking already? a default has been added

Reply:
Sorry, i meant "auto-bombing" and "deprecation warning" or do we plan to support both ?

Reply:
yeah makes sense to add it 👍

Reply:
```suggestion
  void enableDbMigration(String dbMigrationVaultName = "") {
    WarningCollector.addPipelineWarning("deprecated_enable_db_migration_no_vault)", "enableDbMigration() is deprecated, please use enableDbMigration("<vault-name>")", new Date().parse("dd.MM.yyyy", "05.09.2019"))  
```